### ISSUE TITLE
Don't create `/etc/netdata.new` on install.

### ISSUE THREAD

**Issue by Ferroin** [netdata/netdata#8294]
if i install it twice it creates `netdata.new` and changes perms of `netdata`, see

```cmd
test@DEB10-1:/opt/netdata$ ls -l --group-directories-first etc/
total 8
drwx------ 11 netdata netdata 4096 Feb 24 15:06 netdata
drwxr-xr-x 11 netdata netdata 4096 Feb 20 20:56 netdata.new
```

_Originally posted by @ilyam8 in https://github.com/netdata/netdata/issues/8186#issuecomment-590525915_

**Comment by Ferroin** [comment]
Based on investigation, this appears to be an artifact of the code used to migrate from older installs that kept sotck configuration in `/etc/netdata`. I'm not sure _why_ it's getting run outside of those situations though.

**Comment by Ferroin** [comment]
I can't reproduce this, tested in Docker containers for Debian 10, CentOS 7, and Ubuntu 19.04.

@ilyam8 Can you double check that you still get the described behavior and if so provide some more details?

**Comment by ilyam8** [comment]
i used `bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh) --stable-channel --disable-telemetry`. Just install it two times in a row.

**Comment by Ferroin** [comment]
Ah, OK, that's what I missed, you're using the static installer.

That should be fixed in #8315.
